tuf_conformance/test_static_repositories.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+import pytest
+
+from tuf_conformance.client_runner import ClientRunner
+from tuf_conformance.simulator_server import StaticServer
+
+static_repos = []
+for static_dir in os.listdir(os.path.join("tuf_conformance", "static_data")):
```

Comment:
I dont think this works when running the action from another repo, see: https://github.com/sigstore/sigstore-ruby/actions/runs/10457763755/job/28958137257?pr=69

```
_________ ERROR collecting tuf_conformance/test_static_repositories.py _________
../../_actions/theupdateframework/tuf-conformance/07b2952eda3f971c9bade669486643c28bf40a1d/tuf_conformance/test_static_repositories.py:9: in <module>
    for static_dir in os.listdir(os.path.join("tuf_conformance", "static_data")):
E   FileNotFoundError: [Errno 2] No such file or directory: 'tuf_conformance/static_data'
```

Reply:
thanks that seems obvious now that you point it out: #156